src/library/handleHttpResponseError.js

```diff
@@ -28,6 +22,8 @@ const handleHttpResponseError = ({ error, callback, logoutMermaid }) => {
         'A 401 error occurred. handleHttpResponseError requires a logoutMermaid function in its config object.',
       )
     }
+
+    return
```

Comment:
this used to return in the next if statement implicitly. I removed 401 from being part of the next if statement and returned explicitly in the 401-specific if statement.